src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/SetWin32ContextInIDispatchAttributeTests.cs

```diff
@@ -12,6 +12,7 @@ public class SetWin32ContextInIDispatchAttributeTests
 
         [Fact]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/50714", typeof(PlatformDetection), nameof(PlatformDetection.IsBuiltWithAggressiveTrimming), nameof(PlatformDetection.IsBrowser))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/50714", typeof(PlatformDetection), nameof(PlatformDetection.IsAndroidAOT))]
```

Comment:
Same, as for TagToTokenTests.cs, if the failure is the same and known for same underlying reason, can the issue be updated to include that it fails on Android AOT too?

Reply:
it should be the same